### DISCUSSION TITLE
oqtane use cases

### DISCUSSION THREAD

**Discussion by pushkar-irasys** [oqtane/oqtane.framework#1409] [Ideas] [answered]
Hi Oqtane team,

Try to looked on Qutane use cases but didn't find out the details about that.
Just want to know the practical use cases of #oqtane.

1. HR & Payroll 
2. Hospitality Mng system
3. Education Tech apps
4. Accounting Software
etc...

Can we create such type of  applications?

Are there any limitations of Oqtane ?

**Comment by hishamco** [comment, marked as answer]
Why not?

>Are there any limitations of Oqtane ?

Any Framework has limitations, but while its extensible you can roll your own

**Reply by pushkar-irasys** [reply]
Thanks Hisham for this confirmation.

Now we can start new project on Octane.

Understand your response on limitations, and yes we can control that by our own rollout.

**Reply by hishamco** [reply]
So you can start building your external modules, which are then will be pluggable into the Oqtane Framework

**Comment by sbwalker** [comment]
@pushkar-irasys Oqtane is a general purpose modular application framework. It could be used as the foundation for any web application and serve almost any practical use case.
